src/Schema/Event.ts

```diff
@@ -59,6 +60,11 @@ export enum ActionType {
    * Corresponds to {@link TappedFairGroup}
    */
   tappedFairGroup = "tappedFairGroup",
+
+  tappedConsign = "tappedConsign",
+  /**
```

Comment:
This comment should be above the code it describes.

Reply:
Also it corresponds to `TappedConsign` (uppercase) -- the schema this relates to

Reply:
Thanks!